DESCRIPTION

```diff
@@ -90,15 +90,20 @@ Imports:
     tools,
     utils,
     uuid (>= 0.1-4),
-    xml2 (>= 1.3.0)
+    xml2 (>= 1.3.0),
 Suggests:
     knitr,
     mime (>= 0.9),
     reader (>= 1.0.6),
     ritis (>= 1.0.0),
     taxize (<= 0.9.99),
     testthat,
-    worrms (>= 0.4.2)
+    worrms (>= 0.4.2),
+    scales,
```

Comment:
I can do the same as I did with the mapping function to just have the plotting fxn return an informative error message if the required package isn't installed.

Reply:
@clnsmth I can put in conditional statements so scales isn't required, but shouldn't it still be suggested, since the plotting functions will work best with scales?

Reply:
Yes @sokole. Core functions that use suggested packages should check for them and throw a warning (not error) so the function can continue without the suggested package. If a package is required for a function to run through to completion, then it is a dependency.

Having said that, there are some core functions in the current code base that don't work if the suggested package is not available. These need to get fixed, and will be ASAP.

Reply:
Determining what constitutes a core function is a bit of a grey area. It appears to be an acceptable practice to suggest packages if only one function needs it, and if that function doesn't have other functions depending on its output (e.g. `read_data()` is core, `plot_sites()` is not).

Reply:
@clnsmth okay, so we leave the plotting dependencies as "suggests"?

Reply:
@sokole `scales` only affects plot labeling, and therefore any functions using this package should have two modes of operation with/without `scales`.